docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,282 @@
+
+If you are creating an orderer, you obviously do not need to copy the `PeerOUIdentifier` into your `config.yaml` file (or vice versa), but for the sake of simplicity you might want to use the entire section --- extra stanzas do no harm, and they allow the same `config.yaml` to be used for multiple types of nodes and identities associated with an organization.
+
+## Register an identity
+
+While identities that will be used by admins (or other users) and identities used by nodes have different purposes, they are fundamentally all just **identities**: public/private key pairs in which the public key is known to others and the private key is used to sign, generating an output which can be verified to have come from the private key even though the private key itself is never exposed.
+
+As discussed above, an identity is first registered with a CA by a CA admin. This identity is then enrolled by the user of the identity. If you are using the Fabric CA client, this registration command looks like this (regardless of the type of identity you are enrolling, and the type of CA):
+
+```
+./fabric-ca-client register -d --id.name <ID_NAME> --id.secret <ID_SECRET> -u <CA_URL> --mspdir <CA_ADMIN> --id.type <ID_TYPE> --id.attrs $ID_ATTRIBUTE --tls.certfiles <TLSCERT> --csr.hosts $CSR_HOSTNAME
+```
+
+Where the variables are the following:
+
+* `ID_NAME`: The username of the identity. This name will be given to the user out of band, who will use it when enrolling.
+* `ID_SECRET`: The secret (similar to a password) for the identity. Must be given along with the username as part of enrollment.
+* `CA_URL`: The URL of the CA, followed by the port `7054` (unless the default port has been changed).
+* `CA_ADMIN`: The path to the location of the certificates for the admin of the CA the identity is being registered with.
+* `ID_TYPE`: The type (or role) of the identity. There are four possible types: `peer`, `orderer`, `admin`, and `client` (used for applications). This type must be linked to the relevant [NodeOU](#specifiying-nodeous). If NodeOUs are not being used, you can ignore the type and `--id.type` flag.
+* `ID_ATTRIBUTE`: Any attributes specified for this identity. For more information about attributes, check out [Attribute based access control](https://hyperledger-fabric-ca.readthedocs.io/en/latest/users-guide.html#attribute-based-access-control). These attributes can also be added as a JSON array, therefore the `$ID_ATTRIBUTE` is not meant to represent a single attribute but any and all attributes, which should be placed in the register command after the `--id.attrs` flag.
+* `TLSCERT`: The relative path to your the TLS CA root signed certificate (generated when creating the TLS CA).
+* `CSR_HOSTNAME`: Any hostname you wish you use. If you do not specify a CSR hostname, the default hostname, as specified in your CA configuration, will be used.
+
```

Comment:
csr.hosts is only specified on the enroll command, not the register command. Can you remove it from here and the example.